samza-core/src/test/java/org/apache/samza/clustermanager/TestContainerProcessManager.java

```diff
@@ -630,14 +630,15 @@ public void testAllBufferedResourcesAreUtilized() throws Exception {
     config.putAll(getConfigWithHostAffinity());
     config.put("job.container.count", "2");
     config.put("cluster-manager.container.retry.count", "2");
+    config.put("cluster-manager.container.request.timeout.ms", "10000");
```

Comment:
Any reason for this change?
If there are "expiry-based" tests, this will cause build time to increase.